lib/posix-user/exportsyms.uk

```diff
@@ -64,3 +64,6 @@ getgrnam_r
 setgrent
 endgrent
 getgrent
+setgroups
```

Comment:
@mogasergiu 
setgroups is also mentioned on line 62. Shouldn't it be deleted from there?